apps/src/feedback.js

```diff
@@ -501,7 +501,9 @@ FeedbackUtils.prototype.getFeedbackMessage_ = function(options) {
         }
         break;
       case TestResults.TOO_FEW_BLOCKS_FAIL:
-        message = options.level.tooFewBlocksMsg || msg.tooFewBlocksMsg();
+        message = options.level.tooFewBlocksMsg ||
+            options.level.levelIncompleteError ||
+            msg.levelIncompleteError();
```

Comment:
Yeah, the other possibilities I could think of seemed needlessly complicated and/or ugly. Added a comment.